addon/templates/components/polaris-drop-zone.hbs

```diff
@@ -1,60 +1,79 @@
-{{#if showDragOverlay}}
-  <div class="Polaris-DropZone__Overlay">
-    {{#polaris-stack vertical=true spacing="tight"}}
-      {{polaris-icon color="indigo" source=iconDragDrop}}
+{{#with (if label
+  (component "polaris-labelled"
+    id=id
+    label=label
+    labelHidden=labelHidden
+    action=labelAction
+  )
+  (component "empty-wrapper")
+) as |wrapper|}}
+  {{#component wrapper}}
+    <div
+      data-test-drop-zone
+      class={{dropZoneClasses}}
+      aria-disabled={{ariaDisabled}}
+    >
```

Comment:
Just curious as this is a noticeable difference from the React source, is there any reason for not applying the click/drag start actions directly on this element rather than attaching them in the JS file?

Reply:
Ah you're right - I originally left it in the component file because that's where it lived originally, but that was because this element used to be the element generated by the component. Now it's available in the template so yes I can move it there 👍